<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormData</class>
 <widget class="QWidget" name="FormData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>325</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabJoint">
    <attribute name="title">
     <string>JOINT</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tblJoint"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tabTrans">
    <attribute name="title">
     <string>TRANS</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tblTrans"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tabNumber">
    <attribute name="title">
     <string>NUMBER</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tblNumber"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tabString">
    <attribute name="title">
     <string>STRING</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tblString"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
